string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,19 @@
+/* exported capitalizeWord */
+// set variable result
+// lowercase each word (word.toLowerCase())
+// make if statement for JavaScript to uppercase indexes 0 and 4 and slice out unneeded letters; (for JavaScript)
+// set result to toUpperCase index 0 and slice index 1
+
+function capitalizeWord(word) {
+  var result;
+
+  result = word.toLowerCase();
+
+  if (result.charAt(4) === 's') {
+    result = result.charAt(0).toUpperCase() + result.slice(1, 4) + result.charAt(4).toUpperCase() + result.slice(5);
```

Comment:
You can just `return "JavaScript"`, this is a very special case, so you don't need to build the output programmatically